sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -408,6 +408,7 @@ object FunctionRegistry {
     expression[MapValues]("map_values"),
     expression[Size]("size"),
     expression[SortArray]("sort_array"),
+    expression[ConcatArrays]("concat_arrays"),
```

Comment:
Why not reusing `concat`?

concat(array1, array2, ..., arrayN) -> array ?

Reply:
I've already played with this option in my mind, but I'm not sure how concat would be categorized.

Currently, concat is defined as a pure string operation:
  /**
   * @group string_funcs
   * @since 1.5.0
   */
  @scala.annotation.varargs
  def concat(exprs: Column*): Column

Whereas the functionality in this PR belongs rather to the collection_funcs group.

Having just one function for both expressions would be elegant, but can you advise what group should be assigned to concat?

Reply:
How about move it to collection functions?